src/zks.rs

```diff
@@ -305,7 +306,7 @@ mod tests {
             crate::node::ShowVMDetails::None,
             crate::node::ShowGasDetails::None,
             false,
-            false,
+            &system_contracts::Options::BuiltIn,
```

Comment:
nit: We should update this to `let node = InMemoryNode::<HttpForkSource>::default();` so we don't have to update this again in the future. I must've missed this one in the PR where I added the default.

Reply:
ack - will do in a separate cleanup PR.